NorthstarDedicatedTest/logging.cpp

```diff
@@ -473,18 +466,17 @@ void __fastcall Host_Status_PrintClient_Hook(__int64 client, char addresses, voi
 	{
 		active = "spawning";
 	}
-	playerUid = *(unsigned __int64*)(client + 185928);
-	playerSlot = *(DWORD*)(client + 16);
+
 	if (nci)
 	{
-		rate = *(unsigned int*)(nci + 232);
-		loss = (int)(float)((*(float*)(6712 * 1 + nci + 500)) * 100.0);
-		ping = (int)(float)((*(float*)(6712 * 0 + nci + 508)) * 1000.0);
-		connected = COM_FormatSeconds(GetTimeConnected(nci));
+		int rate = *(unsigned int*)(nci + 232);
+		int loss = (int)(float)((*(float*)(6712 * 1 + nci + 500)) * 100.0);
+		int ping = (int)(float)((*(float*)(6712 * 0 + nci + 508)) * 1000.0);
```

Comment:
Is the two-stage cast here needed rather than directly casting it to int?

Reply:
Fixed in 9391bf3da1d048543cbe7a8a7d773ae6e6d83b51